packages/miniflare/src/workers/kv/namespace.worker.ts

```diff
@@ -20,6 +22,7 @@ import {
 	validatePutOptions,
 } from "./validator.worker";
 
+const MAX_BULK_GET_KEYS = 100;
```

Comment:
I was meaning more sharing it with wrangler commands since they should be same and changed at the same time

Reply:
Command doesn't specify the max.

Reply:
delegates to API?

maintaining a constant not ideal but could save a round trip